HeroVault/HeroVault/Features/User/UserFavouritesScreen/UserFavouritesScreenViewModel.swift

```diff
@@ -32,6 +32,10 @@ class UserFavouritesScreenViewModel {
         marvelDataType == .character ? "Characters" : "Comics"
     }
     
+    var emptyMarvelDataList: Bool {
+            return marvelDataListCount == 0
+        }
+    
```

Comment:
not sure if I'm right but I think you can remove the return and use .isEmpty instead of == 0 ?

Reply:
correct me if I'm wrong though 🙃